### ISSUE TITLE
Unable to rename item from library dock. 

### ISSUE THREAD

**Issue by jayrulez** [BearishSun/BansheeEngine#216]
After selecting an item and hitting F2 to rename the idem, I am unable to delete the existing text.

**Comment by jayrulez** [comment]
Oooh. This doesn't seem to be easily reproducible. I'll try y find the exact conditions that triggers this issue.

**Comment by christianclavet** [comment]
(april 2019) 
I'm able to rename the object now with F2, but using the menu, no keyboard input are accepted.
To replicate the issue:
1. Select an object and press the right mouse button to have the context menu appear
2. Select the "rename" item on the list
3. Try to input something, the only accepted key is "ENTER"

If you press "F2" instead when the object is selected, then you can change the name. It almost look like the focus is lost when the context menu appear.

**Comment by BearishSun** [comment]
Hmm, I'm not able to reproduce this one. I can rename using either F2 or context menu. Let me know if you have an idea of what we could be doing differently.

**Comment by christianclavet** [comment]
Confirmed. Tried again on a brand new scene in a new project and it did not happen. Context menu and F2 are working correctly. 

This happen only my old scene and can reproduce anytime with my old test scene. Perhaps something changed and causing problems. Once I knew my old scene was causing the problem, I deleted the "internal" folder and let Banshee rebuild it. Reloaded the project and can rename correctly with the context menu.
